contents/handbook/company/team/tim-glaser.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Tim Glaser's README
+sidebar: Handbook
+showTitle: true
+---
+
+This guide might be helpful in working with me.
+
+## Bio
+
+I've been coding since I've been 11, which isn't as long ago as I'd like it to be. Someone first paid me to write code when I was 13 (though I'm sure they regretted it) and someone else gainfully employed me when I was 16.
+
+Originally from the Netherlands, though I quickly moved to London (I do not generally enjoy nice weather) where I joined Arachnys and shortly met James Hawkins. I went from being a software engineer, to product manager, to "leading" an R&D team, which consisted of just me.
+
+After four years I thought it was time to go do something else and had lined up a new job. Roughly 37 seconds after it was announced James wanted to "grab a beer." While plying me with alcohol, he convinced me to give up this fancy new job and instead start a startup with him.
+
+In my 'spare' time, I fall down snowy mountains, wrestle in the mud over an egg-shaped ball and watch a lot of Bondi beach in order to perfect my Australian accent.
+
+## Areas of responsibility
+
+- Make sure we move fast
+- Make sure the engineering team has all the context it needs
+- Make sure the engineering team is happy
+- Make sure we're building in the right direction
+- Sounding board to James Hawkins (CEO)
+
+## Quirks
+
+- I come out with opinions quickly and strongly. They are actually weekly held so please push back when you disagree, I'll take it well.
```

Comment:
"weakly" or every week?

Reply:
strong opinions held for one week at a time